src/app/app.module.ts

```diff
@@ -68,7 +69,8 @@ import { DownloadQueModule } from './modules/download-que/download-que.module';
     CardClusterModule,
     TicketModule,
     SearchFieldModule,
-    DownloadQueModule
+    DownloadQueModule,
```

Comment:
What is DownloadQueueModule?

Reply:
I see where Jack started this. Probably want to remove the include for now.

Reply:
I can remove this in a later commit